medusa/server/web/home/handler.py

```diff
@@ -201,6 +201,10 @@ def testSABnzbd(host=None, username=None, password=None, apikey=None):
 
     @staticmethod
     def testNZBget(host=None, username=None, password=None, use_https=False):
+        if use_https == 'false':
+            use_https = False
+        else:
+            use_https = True
         connected_status = nzbget.testNZB(host, username, password, use_https)
```

Comment:
The current values are 'on' 'off', they're now 'true' 'false'. Python sees all 4 values as truthy values and sets it as true. 😞